src/config/config.go

```diff
@@ -18,6 +18,8 @@ type Config struct {
 	OpenshiftVersion string
 	Hostname         string
 	ControllerImage  string
+	PullSecretPath   string
+	PullSecretToken  string
```

Comment:
Maybe this does not need to be a global?
Maybe it should be part of the ConrollerConfig?

Reply:
It is needed for creating the BM Client in assisted-installer, and for creating the k8s secret for assisted-installer controller.

Reply:
Done